### PR TITLE
Documentation for the GitOps 1.11.0 release notes.

### PR DESCRIPTION
**Pull request by LunguTheEngineer**
- Jira issue: [RHDEVDOC-5680](https://issues.redhat.com/browse/RHDEVDOCS-5680)
- Cherry-pick branches: gitops-docs-1.11
- OCP  team: Devtools outerloop
- Doc preview: [GitOps 1.11.0 Release Notes](https://68509--docspreview.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes#gitops-release-notes-1-11-0_gitops-release-notes)
- SME reviews by: @svghadi , @reginapizza , @jaideepr97 
- QE reviews by: @varshab1210 
- Peer reviews by: @Srivaralakshmi

### CODE REVIEW

**Review by svghadi — APPROVED** [review]
/lgtm
Thanks @masauso-lungu

File: modules/gitops-release-notes-1-11-0.adoc
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-0.adoc:28]: @masauso-lungu Do we have any security advidory this release? If not we can remove this section.

Please clarify with the SMEs. Thank you!

File: modules/gitops-release-notes-1-11-0.adoc
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-0.adoc:33]: Suggestion: Rewrite for clarity. What is `argocd-tls-certs-cm`? A file or config map? Please mention that wherever it comes in content.

File: modules/gitops-release-notes-1-11-0.adoc
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-11-0.adoc:35]:
```suggestion
* Before this update, users would face `PrometheusOperatorRejectedResources` alerts when the {gitops-title} Operator was not installed in the `openshift-gitops-operator` namespace. The problem affected users who upgraded from earlier versions of the {gitops-title} Operator to v1.10. This update fixes the issue by updating the Operator's `serverName` metrics service to reflect the correct installation namespace. Now, users who upgrade or install the {gitops-title} Operator in namespaces other than `openshift-gitops-operator` should not see these alerts. link:https://issues.redhat.com/browse/GITOPS-3424[GITOPS-3424] 
```

**Review by Srivaralakshmi — COMMENTED** [review]
@masauso-lungu Thanks a lot for the effort and work put in! PTAL at the suggestions.

File: modules/gitops-release-notes-1-11-0.adoc
Comment (svghadi) [reply modules/gitops-release-notes-1-11-0.adoc:33]: It's a configmap.

File: modules/gitops-release-notes-1-11-0.adoc
Comment (reginapizza) [reply modules/gitops-release-notes-1-11-0.adoc:28]: It's just a regular enhancement advisory this time!

Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:28]: The available errata is an enhancemnt and i was due to update it. @reginapizza, do we need this RHEA published as we normally do with RHSA?, thank you

Comment (reginapizza) [reply modules/gitops-release-notes-1-11-0.adoc:28]: I'm not sure of the question @masauso-lungu... would this not be a question for @Srivaralakshmi ?

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:35]: ACK.

Addressed.

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:33]: ACK.

Resolved.

File: modules/gitops-release-notes-1-11-0.adoc
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-11-0.adoc:28]: @masauso-lungu AFAIU, we only have a regular enhancement advisory this time! If there is no security advisory, then please drop this section. I hope this answer clarifies the confusion.

Thanks, @reginapizza for the response. This is helpful.

Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:28]: Thank you @Srivaralakshmi . I will drop it.

File: modules/go-compatibility-and-support-matrix.adoc
Comment (masauso-lungu) [reply modules/go-compatibility-and-support-matrix.adoc]: updated. @jaideepr97

File: modules/gitops-release-notes-1-11-0.adoc
Comment (varshab1210) [inline modules/gitops-release-notes-1-11-0.adoc:17]:
```suggestion
To disable a component, set the `enabled` flag to `false` in the `.spec.<component>.enabled` field of the Argo CD Custom Resource (CR). link:https://issues.redhat.com/browse/GITOPS-3723[GITOPS-3723]
```

Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc]: ACK.
Addressed

**Comment by varshab1210** [comment]
LGTM, thanks

File: modules/gitops-release-notes-1-11-0.adoc
Comment (varshab1210) [reply modules/gitops-release-notes-1-11-0.adoc:16]: This s not required as it was working as expected with openshift-gitops instance

Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:16]: Great. Thank you @varshab1210

File: modules/gitops-release-notes-1-11-0.adoc
Comment (xenolinux) [inline modules/gitops-release-notes-1-11-0.adoc:41]:
```suggestion
* Before this update, users would face `PrometheusOperatorRejectedResources` alerts when the {gitops-title} Operator was not installed in the `openshift-gitops-operator` namespace. The problem affected users who upgraded from earlier versions of the {gitops-title} Operator to v1.10. This update fixes the issue by updating the Operator's `serverName` metrics service to reflect the correct installation namespace. Now, users who upgrade or install the {gitops-title} Operator in namespaces other than `openshift-gitops-operator` cannot see these alerts. link:https://issues.redhat.com/browse/GITOPS-3424[GITOPS-3424]
```

Comment (xenolinux) [reply modules/gitops-release-notes-1-11-0.adoc:41]: Can we use "cannot" instead of "should not"? I think "cannot" provides clarity

File: modules/gitops-release-notes-1-11-0.adoc
Comment (xenolinux) [inline modules/gitops-release-notes-1-11-0.adoc:21]: I suggest to consider adding initial fields to this YAML. For example, some more context like: `apiVersion: argoproj.io/v1alpha1
kind: ArgoCD
metadata:
  name: example-argocd`

File: modules/gitops-release-notes-1-11-0.adoc
Comment (xenolinux) [inline modules/gitops-release-notes-1-11-0.adoc:19]:
```suggestion
For example:
```

File: modules/gitops-release-notes-1-11-0.adoc
Comment (xenolinux) [inline modules/gitops-release-notes-1-11-0.adoc:39]:
```suggestion
* Before this update, the Argo CD Notifications Controller did not support custom certificates added to the `argocd-tls-certs-cm` config map. As a result, notification services with custom certificates did not receive notifications due to the "x509: certificate signed by unknown authority" error. This update fixes the issue by correctly initializing the cert resolver function in the Argo CD Notifications Controller to load all certificates stored in the `argocd-tls-certs-cm` config map. Now, notification services with custom certificates can successfully receive notifications. link:https://issues.redhat.com/browse/GITOPS-2809[GITOPS-2809]
```

**Review by xenolinux — COMMENTED** [review]
I left a few suggestions but otherwise looks good!
Please make sure to squash the commits.

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

File: modules/gitops-release-notes-1-11-0.adoc
Comment (abhatt-rh) [inline modules/gitops-release-notes-1-11-0.adoc:5]: See OpenShift documentation guidelines for [module file metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)
```suggestion
:_mod-docs-content-type: REFERENCE
```

File: modules/gitops-release-notes-1-11-0.adoc
Comment (abhatt-rh) [inline modules/gitops-release-notes-1-11-0.adoc:20]: Per OpenShift documentation [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output:~:text=do%20not%20insert%20a%20space):
> do not insert a space in between the [source] tag and the metadata.

```suggestion
[source,yaml]
```

File: modules/gitops-release-notes-1-11-0.adoc
Comment (abhatt-rh) [inline modules/gitops-release-notes-1-11-0.adoc:31]: See Red Hat supplementary style guide for [Future releases or plans](https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future)

> Do not promise that a feature or a fix for a known issue will be included in an upcoming release or according to a specific timeline.

```suggestion
This feature is currently limited to the `redis` and `application-controller` components. It is expected that support for other components will be included in a future {gitops-title} release.
```

File: modules/gitops-release-notes-1-11-0.adoc
Comment (abhatt-rh) [inline modules/gitops-release-notes-1-11-0.adoc:39]:
```suggestion
* Before this update, the Argo CD Notifications Controller did not support custom certificates added to `argocd-tls-certs-cm` config map. As a result, notification services with custom certificates did not receive notifications due to the `x509: certificate signed by unknown authority` error message. This update fixes the issue by correctly initializing the cert resolver function in the Argo CD Notifications Controller to load all certificates stored in `argocd-tls-certs-cm` config map. Now, notification services with custom certificates can successfully receive notifications. link:https://issues.redhat.com/browse/GITOPS-2809[GITOPS-2809]
```

**Review by abhatt-rh — COMMENTED** [review]
@masauso-lungu and @xenolinux: Peer review comments mostly LGTM!
I added a few more suggestions for your consideration; also adding a reminder here  to squash your commits before requesting a merge review. Thanks

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:45]: ACK.
addressed

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:39]: ACK.
Addressed

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:31]: ACK.
Addressed

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:41]: IMO I feel "should not" sounds more natural and is more suitable for the described situation.

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:5]: ACK.

Addressed

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:19]: ACK.

Addressed

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:39]: ACK.
Addressed.

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:20]: ACK.
Addressed

File: modules/gitops-release-notes-1-11-0.adoc
Comment (masauso-lungu) [reply modules/gitops-release-notes-1-11-0.adoc:21]: ACK.

Addressed.

**Pull request merged**
